core/src/main/java/org/lflang/generator/c/CPreambleGenerator.java

```diff
@@ -87,13 +87,12 @@ public static String generateDefineDirectives(TargetConfig targetConfig, Path sr
     //         targetConfig.clockSyncOptions
     //     ));
     // }
-
-    if (targetConfig.get(ThreadingProperty.INSTANCE)) {
+    if (!targetConfig.get(SingleThreadedProperty.INSTANCE)) {
       definitions.put("LF_THREADED", "1");
     } else {
       definitions.put("LF_UNTHREADED", "1");
```

Comment:
I guess these will have to be updated to work with the reactor-c changes?

Reply:
Ah, thanks for spotting this. Those changes got lost somehow. I'll reintroduce them.